crates/sui-config/tests/snapshots/snapshot_tests__genesis_config_snapshot_matches.snap

```diff
@@ -12,7 +12,7 @@ parameters:
   stake_subsidy_initial_distribution_amount: 1000000000000000
   stake_subsidy_period_length: 30
   stake_subsidy_decrease_rate: 10000
-committee_size: 4
+committee_size: 0
```

Comment:
why is this 0?

Reply:
resolved offline

Reply:
Changing the default value to 0, so it is possible to differentiate when `committee_size` is set. When `committee_size` is set explicitly, it should match the number of validator info if that list is also set.